plugins/palo_alto_cortex_xdr/plugin.spec.yaml

```diff
@@ -4,8 +4,8 @@ products: [insightconnect]
 name: palo_alto_cortex_xdr
 title: Palo Alto Cortex XDR
 description: Stop modern attacks with the industry's first extended detection and response platform that spans your endpoints, network and cloud data
-version: 2.1.1
-supported_versions: ["Palo Alto Cortex XDR API v1"]
+version: 2.2.0
+supported_versions: ["2022-03-28 Palo Alto Cortex XDR API v1"]
```

Comment:
Is this the new way of doing things in the past week?
`2022-03-28 Palo Alto Cortex XDR API v1`
If nothing has been said about this, I think it would be safer to leave this just as 
`2022-03-28`